test/journeys/specs/smoke/widget-recents/space-type-filter.js

```diff
@@ -0,0 +1,190 @@
+import {assert} from 'chai';
+
+import {createSpace, disconnectDevices, registerDevices, setupGroupTestUsers} from '../../../lib/test-users';
+
+import {jobNames, renameJob, updateJobStatus} from '../../../lib/test-helpers';
+import {elements} from '../../../lib/test-helpers/recents-widget';
+
+describe('Widget Recents with Space Type Filter', () => {
+  const browserLocal = browser.select('browserLocal');
+  const TIMEOUT = 12000;
+  const RETRY_TOTAL = 3;
```

Comment:
I like the idea of retries, but can we make it configurable?

Reply:
It is configurable in wdio.conf.js under mocha options:

<img width="227" alt="screen shot 2018-12-19 at 11 09 32 am" src="https://user-images.githubusercontent.com/2557321/50242262-a75a8200-037e-11e9-9b52-3c5d7adbf544.png">

It will apply globally though and in order for your test block to work, it has to be normal function and not a fat arrow one.

An alternative is setting retries within a suite and/or individual tests: http://webdriver.io/guide/testrunner/retry.html ...